demonstrations_categories.json

```
@@ -0,0 +1,37 @@
+[
+    {
+        "title": "Getting Started",
+        "urlFragment": "getting-started",
+        "description": "Begin your journey into quantum computing using PennyLane."
+    },
+    {
+        "title": "Optimization",
+        "urlFragment": "optimization",
+        "description": "Learn how to train quantum circuits like neural networks, using the latest tips and tricks from the literature."
+    },
+    {
+        "title": "Quantum Machine Learning",
+        "urlFragment": "quantum-machine-learning",
+        "description": "Explore cutting-edge research in quantum machine learning using PennyLane."
+    },
+    {
+        "title": "Quantum Chemistry",
+        "urlFragment": "quantum-chemistry",
+        "description": "Study the properties of molecules and materials using quantum computing."
+    },
+    {
+        "title": "Devices and Performance",
+        "urlFragment": "devices-and-performance",
+        "description": "Here you will discover and learn how to use many of the devices offered by PennyLane. From built in devices and its different interfaces to external devices."
```

```suggestion
        "description": "Here you will discover and learn how to use many of the devices offered by PennyLane. From built-in devices and their different interfaces to external devices."
```
Small correction. But this tagline doesn't really pack a punch. The new descriptions also don't match the ones from the 'old' categories.

Maybe we can revisit the descriptions here once all the demos have been packed into their drawers?

---

Yep - we can update these as often as we like - the changes should propagate through to the live React site.